frontend/forms/OBSBasic.ui

```diff
@@ -34,7 +34,7 @@
    <string notr="true"/>
   </property>
   <property name="dockOptions">
-   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
+   <set>QMainWindow::DockOption::AllowNestedDocks|QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
```

Comment:
I think I was still avoiding fully-qualified enums until we only support platforms that use Qt 6.5+, because the uic changes were only ported back to 6.5. Unless someone can assure me that this does not negatively affect Ubuntu/Linux builds (where we still have to worry about Qt 6.4 being present), I'd prefer to continue avoiding fully qualified enums, to my own dismay.